src/main/java/uk/gov/hmcts/reform/ccd/document/am/util/SecurityUtils.java

```diff
@@ -20,8 +22,24 @@ public SecurityUtils(final AuthTokenGenerator authTokenGenerator) {
     public HttpHeaders authorizationHeaders() {
         final HttpHeaders headers = new HttpHeaders();
         headers.add(SERVICE_AUTHORIZATION, authTokenGenerator.generate());
+
+        if (SecurityContextHolder.getContext().getAuthentication() != null) {
+            final ServiceAndUserDetails serviceAndUser = (ServiceAndUserDetails) SecurityContextHolder.getContext().getAuthentication().getPrincipal();
+            if (serviceAndUser.getPassword() != null) {
+                headers.add(HttpHeaders.AUTHORIZATION, serviceAndUser.getPassword());
```

Comment:
why are we setting the password for the authosization header?

Reply:
Password is user token only.